installer.tpl.sh

```diff
@@ -122,6 +125,12 @@ setup_type ()
 
 setup_ver ()
 {
+  ARCH=$(uname -m)
```

Comment:
It seems L245 can be removed now, right?

Reply:
yes, done